content/en/cloudcraft/getting-started/live-vs-snapshot-diagrams.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Live vs Snapshot Diagrams
+---
+
+## Overview
+
+Cloudcraft offers two types of diagrams for visualizing your cloud infrastructure: Live and Snapshot. This document explains the key differences between these diagram types and how to use them effectively.
+
+## Live diagrams
+
+Live diagrams provide real-time representation of your AWS or Azure infrastructure. Key features include:
+
+- **Automatic updates**: The diagram reflects changes in your infrastructure when you access it.
+- **Real-time accuracy**: Ensures your diagram always represents the current state of your cloud infrastructure.
+- **Read-only**: Prevents accidental modifications to your infrastructure layout.
```

Comment:
Because the user can modify their diagram with the caveat that we automatically switch them to snapshot mode, I wonder if "read-only" is a bit confusing / the right term to describe this functionality? You could still accidentally modify a live diagram, and would need to undo your modification to get the diagram back to its live state.

Reply:
Hm, true. Any suggestion for a new name?

Reply:
I feel like `Real-time accuracy` captures the idea that no manual edits are in the diagram. Maybe the third bullet can mention `Live filtering` instead, as in the diagram will always be generated based on the user's filters and group bys?